libs/ledger-live-common/src/families/evm/createTransaction.ts

```diff
@@ -18,7 +18,7 @@ export const createTransaction: AccountBridge<EvmTransaction>["createTransaction
     maxPriorityFeePerGas: new BigNumber(0),
     gasLimit: new BigNumber(21000),
     nonce: account.operationsCount + 1,
-    chainId: account.currency?.ethereumLikeInfo?.chainId || 0,
+    chainId: (account as Account).currency?.ethereumLikeInfo?.chainId || 0,
```

Comment:
- [live-common](https://ledgerhq.atlassian.net/browse/LIVE-common) is outside the scope of this PR
- We put as Account for a reason to prevent issues dur to the nature of AccountLike